dali/pipeline/operators/fused/crop_mirror_normalize.cu

```diff
@@ -55,7 +55,7 @@ __global__ void BatchedCropMirrorNormalizePermuteKernel(
             int in_idx = c + C*mirrored_width + in_step*h;  // HWC, mirrored
             int out_idx = c*H*W + h*W + w;  // CHW
 
-            output_ptr[out_idx] = static_cast<Out>(
+            output_ptr[out_idx] = StaticCastGpu<Out>( // static_cast<Out>(
```

Comment:
Remove static_cast<Out> instead commenting it out.

Reply:
Ok, I will follow this. And I still working on lint. 
Thank you for reviewing